docs-src/concepts/what-is-a-cloud-account-id.md

```diff
@@ -8,12 +8,13 @@ tags:
   - explanation
 ---
 
-A Temporal Cloud Account Id is a unique identifier for a customer for the entire time they use Temporal Cloud.
-Temporal Technologies assigns each Account Id, which is an opaque code of five or six alphanumeric characters, such as `f45a2`.
-The Account Id is shown in the upper-right corner of Workflows and Schedules listed in the Web UI.
+A Temporal Cloud Account Id is a unique customer identifier assigned by Temporal Technologies.
+The Id is a short string of numbers and letters, at least five characters long.
+The identifier remains constant for the entire time a customer uses Temporal Cloud.
 
-:::info
+To find an Account Id, login to Temporal Cloud.
+Navigate to the Namespace list page from the left-side vertical navigation.
+The Account Id is the alphanumeric character string found after the period in any Cloud Namespace name.
 
-For more information on obtaining a Temporal Cloud Account Id, see our [Cloud account commands](/cloud/tcld/account/get) page.
-
-:::
+To view Namespace information from the command line instead of the Web, use the `tcld` CLI utility.
```

Comment:
If we add this info, we should also add the command here. This is where the link in the info box is useful, provided it links to the correct command.

Reply:
It is already added on line 20.  
![image](https://github.com/temporalio/documentation/assets/156466156/c2bf66df-d58d-494f-ac2d-b6b4cf49a30d)

Reply:
ah my mistake!